docs/PowerAuth-Enrollment-Server-1.7.0.md

```diff
@@ -0,0 +1,13 @@
+# Migration from 1.6.x to 1.7.x
+
+This guide contains instructions for migration from PowerAuth Enrollment Server version `1.6.x` to version `1.7.0`.
+
+
+## REST API
+
+
+### Register for Push Messages (Token)
+
+The endpoint `POST /api/push/device/register/token` now strictly validates `platform` against values `ios`, `android` or `huawei`.
+If you use the SKD, you should not be affected.
+Unless you are calling that manually.
```

Comment:
I would skip the last sentence. 
Or use:
In case you are not using PowerAuth SKD you need to ensure the proper value of `platform` when calling the push device registration endpoint.